jobs/rocoto/atmanalpost.sh

```diff
@@ -1,10 +1,12 @@
 #! /usr/bin/env bash
 
+export STRICT="NO"
```

Comment:
Why are you turning strict off in all these? Especially to just turn it back on immediately after (though that doesn't really do anything unless you call `set_strict` later).

Reply:
I neither like nor advocate these lines.   They are, for the time being, necessary due to the problems documented in issue #1067.   If @aerorahul or you have a better solution which allows us to cycle UFS-DA on Orion, please let us know.

Reply:
Okay, I don't want to hold up this PR. But, if you can provide instructions on how to set up an experiment with UFSDA, I'll see if I can figure out what's going on.

Reply:
It's related (99% sure) to module purge/module load with the conda python env we are using. It is not a good/permanent solution. @WalterKolczynski-NOAA do you have a workflow python env that works properly that we can install needed utilities in?